3.Tests/UI/StepDefinitions/User/LogoutSteps.cs

```diff
@@ -0,0 +1,42 @@
+﻿using SeleniumTest.Core.Drivers;
+using SeleniumTest.Tests.Steps.Commons;
+using TechTalk.SpecFlow;
+using Views.WebAppPages;
+
+namespace SeleniumTest.Tests.Steps.User;
```

Comment:
Pages and Tests projects should have the AUT prefixed on the namespace: `namespace todoly.Tests.Steps.User;`